src/Menu.php

```diff
@@ -121,6 +121,8 @@ public function addGroup($name, string $template = 'menugroup.html')
 
         if (isset($name['icon'])) {
             Icon::addTo($group, [$name['icon']], ['Icon'])->removeClass('item');
+        } elseif (isset($name['label'])) {
+            Label::addTo($group, [$name['label']], ['Icon'])->removeClass('item')->addClass('mini');
```

Comment:
you say this is supported somewhere and somewhere not - can we dedup the code at least with private method?